_data/related-jobs.yml

```diff
@@ -1,3 +1,8 @@
+- expires: 2024-12-04
+  location: New York University, New York, NY (mostly remote - more than 60% of time)
```

Comment:
Minor suggestion: `mostly remote - more than 60% of time` -> `mostly remote: remote more than 60% of time`

Reply:
I think it's ok as is